Sources/EpoxyCore/SwiftUI/EpoxySwiftUIHostingView.swift

```diff
@@ -62,14 +62,19 @@ public final class EpoxySwiftUIHostingView<RootView: View>: UIView, EpoxyableVie
   public init(style: Style) {
     // Ignore the safe area to ensure the view isn't laid out incorrectly when being sized while
     // overlapping the safe area.
+    epoxyContent = EpoxyHostingContent(rootView: style.initialContent.rootView)
     viewController = EpoxySwiftUIHostingController(
-      rootView: .init(environment: epoxyEnvironment, content: style.initialContent.rootView),
+      rootView: .init(content: epoxyContent, environment: epoxyEnvironment),
       ignoreSafeArea: true)
 
     dataID = style.initialContent.dataID ?? DefaultDataID.noneProvided as AnyHashable
 
     super.init(frame: .zero)
 
+    epoxyEnvironment.epoxyIntrinsicContentSizeInvalidator = .init(invalidate: {
+      [weak self] in
```

Comment:
nit: we typically don't wrap these
```suggestion
    epoxyEnvironment.epoxyIntrinsicContentSizeInvalidator = .init(invalidate: { [weak self] in
```